psi4/run_psi4.py

```diff
@@ -133,6 +133,9 @@
     data_dir = os.path.abspath(os.path.expanduser(args["psidatadir"]))
     os.environ["PSIDATADIR"] = data_dir
 
+if "@ENABLE_PCMSolver@" in ["ON", "YES", "TRUE"]:
+    sys.path.insert(1, "@PCMSolver_PYMOD@")
+
```

Comment:
Thorough test for above queued for next PR. And after https://github.com/PCMSolver/pcmsolver/issues/40, I'll see about getting PCMSolver working with API mode.